gist/src/main/java/build/gist/presentation/GistView.kt

```diff
@@ -70,6 +71,7 @@ class GistView @JvmOverloads constructor(
                                 shouldLogAction = false
                                 Log.i(GIST_TAG, "Dismissing from action: $action")
                                 GistSdk.handleGistClosed(message)
+                                GistSdk.removePersistentMessage(message)
```

Comment:
Instead of this function call, should we instead handle the logic of closing a persistent message in `GistSdk.handleGistClosed(message)`? 

Otherwise, I would think that anytime we want to close an in-app message, we have to remember to always call 2 functions: 

```kt
GistSdk.handleGistClosed(message)
GistSdk.removePersistentMessage(message)
```

Reply:
We don't always want to remove the persistent message on close, there's a use case where a message will dismiss itself and open another modal.

We decided to remove a persistent message from the queue only when either the developer or end customer expressively chooses to do so.

To your point of always remembering, funnily enough, I forgot to add it to the manual dismissal method. So to remedy this I added handleGistClosed into removePersistentMessage and its now called `dismissPersistentMessage`